sdk/core/azure-core-test/src/test/java/com/azure/core/test/TestProxyTests.java

```diff
@@ -414,16 +414,49 @@ public void testRecordWithRedirect() {
     }
 
     private RecordedTestProxyData readDataFromFile() {
-        String filePath = Paths.get(TestUtils.getRecordFolder().getPath(), this.testContextManager.getTestPlaybackRecordingName()) + ".json";
-
-        File recordFile = new File(filePath);
-        try (BufferedReader reader = Files.newBufferedReader(recordFile.toPath())) {
+        try {
+            File recordFile = new File(locateAssetJsonFilePath());
+            BufferedReader reader = Files.newBufferedReader(recordFile.toPath());
```

Comment:
Just FYI we can't always assume that `assets.json` is present right? We only want to pass along assets.json path if it actually exists. This code still assumes it'll always be there.

(to be clear, that's not an explanation for the current failure, just for the future.

Reply:
Since this is a core-test package I think it should be fine to have this hard-coded here.